src/confcom/azext_confcom/container.py

```diff
@@ -278,6 +278,109 @@ def extract_user(container_json: Any) -> Dict:
                 config.POLICY_FIELD_CONTAINERS_ELEMENTS_USER_PATTERN: str(run_as_group_value),
                 config.POLICY_FIELD_CONTAINERS_ELEMENTS_USER_STRATEGY: "id"
             }
+
+        # get the field for privileged
+        privileged_value = case_insensitive_dict_get(
```

Comment:
you should make a separate function for extracting capabilities, this function is for the user field